silver/tests/spec/test_payment_method.py

```diff
@@ -0,0 +1,59 @@
+from rest_framework import status
+from rest_framework.renderers import JSONRenderer
+from rest_framework.reverse import reverse
+from rest_framework.test import APITestCase
+
+from silver.api.serializers import PaymentMethodSerializer
+from silver.models import PaymentMethod
+from silver.tests.factories import AdminUserFactory, CustomerFactory, \
+    PaymentMethodFactory
+
+
+class TestPaymentMethodEndpoints(APITestCase):
+    def setUp(self):
+        admin_user = AdminUserFactory.create()
+        self.client.force_authenticate(user=admin_user)
+
+    def test_filter_processor(self):
+        customer = CustomerFactory.create()
+        payment_method = PaymentMethodFactory.create(
+            customer=customer
+        )
+
+        url = reverse('payment-method-list', kwargs={
+            'customer_pk': customer.id
+        })
+
+        good_url = url + '?processor=manual'
+        bad_url = url + '?processor=random'
+
+        self._assert_get_data(good_url, [payment_method])
+        self._assert_get_data(bad_url, [])
```

Comment:
It is a listing page, a listing page if it has no content it will produce an empty listing.